bot/middlewares/any_middleware.py

```diff
@@ -14,6 +14,10 @@
     TelegramObject,
 )
 
+from bot.database.database_manager import DatabaseManager
+from bot.responses.bot_message_handler_responses import get_limit_exceeded_message
+from bot.settings import settings
+
 
 class AnyMiddleware(BaseMiddleware):
```

Comment:
```suggestion
class AnyMiddleware(BotMiddleware):
```

mamy wlasna klase bazowa